chromadb/test/test_api.py

```diff
@@ -1361,3 +1362,14 @@ def test_invalid_embeddings(api):
     with pytest.raises(ValueError) as e:
         collection.upsert(**invalid_records)
     assert "embedding" in str(e.value)
+
```

Comment:
we treat test_api.py as deprecated. lets break this out into its own test

Reply:
It just occurred to me that I need to add a generator for Auth Providers instead of having many fixtures for each provider with each config type (e.g. file, env, params)

Reply:
Scratch that! 

![image](https://github.com/chroma-core/chroma/assets/1157440/ac41a4ec-b29e-44be-aaaf-384713b38ce4)

Reply:
hahahahahahahahahahaah